pilot/pkg/bootstrap/configcontroller.go

```diff
@@ -125,7 +125,8 @@ func (s *Server) initMCPConfigController(args *PilotArgs) error {
 	var conns []*grpc.ClientConn
 	var configStores []model.ConfigStoreCache
 
-	s.mcpOptions = &mcp.Options{
+	mcpOptions := &mcp.Options{
+		ClusterID:    s.clusterID,
```

Comment:
why change this?

Reply:
We should pass the clusterID

Reply:
why? I don't feel comfortable adding random things without understanding what is being fixed

Reply:
I think we missed setting this value, by default it is `Kubernetes`

Reply:
Ah, just look again, it seems this field is not used by mcp controller, to clean code i should remove it as well.